autogen.sh

```diff
@@ -42,7 +42,6 @@ fi
 
 autoreconf -ivf
 
-echo "./configure $@"
+echo "./configure $*"
 chmod +x configure
-./configure $@
-
+./configure "$@"
```

Comment:
Does `./autogen.sh --with-local-libgcrypt --with-pcre2` still work?

Reply:
As far as I can tell, yes. I managed to successfully build the project with these parameters just now.

As [SC2068](https://github.com/koalaman/shellcheck/wiki/SC2068) says:
> Let's say you have four arguments: `baz`, `foo bar`, `*` and `/*/*/*/*`
>
> "$@" will expand into exactly that: `baz`, `foo bar`, `*` and `/*/*/*/*`
>
> $@ will expand into multiple other arguments: `baz`, `foo`, `bar`, `file.txt`, `otherfile.jpg`, and (eventually) a list of most files on the system

So it should only affect whitespace and globbing for each argument separately.